src/components/Review.jsx

```diff
@@ -36,14 +59,14 @@ const Review = () => {
       ></img>
       <h3 className="review-creator">{review.designer}</h3>
       <p className="creation-time">{formattedDate}</p>
-      <button className="vote-container">
+      {err ? <p>{err}</p> : null}
+      <button className="vote-container" onClick={handleClick}>
         <img src={likeButton} alt="like button" className="thumb-icon" />
-        <p>{review.votes}</p>
+        <p className={isClicked ? "clicked" : ""}>{voteCount}</p>
       </button>
       <p className="review-body">{review.review_body}</p>
       <div className="comments-container">
         <p>{review.comment_count} Comments</p>
-        {}
       </div>
     </section>
   );
```

Comment:
Good optimistic rendering here, the button is intuitive and the toggle on and off works nicely . Nice to see async await implemented.